spec/export/index.md

```diff
@@ -20,7 +20,7 @@ The scope of this document does NOT include:
 
 * Newline-delimited JSON.  [http://ndjson.org](http://ndjson.org)
 * The OAuth 2.0 Authorization Framework, RFC6749, [https://tools.ietf.org/html/rfc6749](https://tools.ietf.org/html/rfc6749)
-* HL7 FHIR Release 3, [https://www.hl7.org/fhir/stu3/](https://www.hl7.org/fhir/stu3/)
+* HL7 FHIR Release 4, [https://www.hl7.org/fhir/stu3/](https://www.hl7.org/fhir/stu3/)
```

Comment:
@chris1uphealth please just change this to say HL7 FHIR [https://www.hl7.org/fhir/](https://www.hl7.org/fhir/)